Author: James Westby <james.westby@canonical.com>
Description:
 Adding patch define IXP_NEEDAPI to 88 so that we get the ixp API that is
 required. File re-written in hg upstream to use the new API (Closes: #511954).

diff -Naurp wmii.orig/cmd/wmii/dat.h wmii/cmd/wmii/dat.h
--- wmii.orig/cmd/wmii/dat.h	2009-08-04 10:37:35.000000000 +0000
+++ wmii/cmd/wmii/dat.h	2009-08-04 13:58:48.000000000 +0000
@@ -5,6 +5,7 @@
 #define _XOPEN_SOURCE 600
 #define IXP_P9_STRUCTS
 #define IXP_NO_P9_
+#define IXP_NEEDAPI 88
 #include <regexp9.h>
 #include <stdint.h>
 #include <ixp.h>
diff -Naurp wmii.orig/cmd/wmiir.c wmii/cmd/wmiir.c
--- wmii.orig/cmd/wmiir.c	2009-08-04 10:37:35.000000000 +0000
+++ wmii/cmd/wmiir.c	2009-08-04 13:58:48.000000000 +0000
@@ -3,6 +3,7 @@
  */
 #define IXP_NO_P9_
 #define IXP_P9_STRUCTS
+#define IXP_NEEDAPI 88
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
